deploy/helm/rocketride/templates/hpa.yaml

```
@@ -0,0 +1,33 @@
+  labels:
+    {{- include "rocketride.labels" . | nindent 4 }}
+    app.kubernetes.io/component: engine
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "rocketride.fullname" . }}-engine
+  minReplicas: {{ .Values.engine.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.engine.autoscaling.maxReplicas }}
+  metrics:
+    {{- if .Values.engine.autoscaling.targetCPUUtilizationPercentage }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)